src/services/recipient.js

```diff
@@ -652,7 +662,7 @@ export async function getGoalsByActivityRecipient(
         model: GoalFieldResponse,
         as: 'responses',
         required: false,
-        attributes: ['response', 'goalId'],
+        attributes: ['response', 'goalId', 'goalTemplateFieldPromptId'],
```

Comment:
now that the isFei is in the command, is this needed here?

Reply:
removed.